### DISCUSSION TITLE
Add `attributes` as code cells option (alongside `classes`)

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#5587] [Feature Requests]
From #5456 and https://github.com/yihui/knitr/issues/2256, I believe it would be very useful to be able to add any kind of attributes and not only classes to cell container.
For instance, `knitr` offers `class-*` and the general `attr-*` options to customise code cells.

This `attributes` could help leverage many Bootstrap features already embedded in Quarto, such as popover or tooltip which uses `data-bs-*` attributes.

**Comment by mcanouil** [comment]
@dragonstyle (cc @cscheid) do you think that's something easily achievable? I believe it could be a nice complementary/general option to `classes` which will completely avoid the need to put code cell into fenced divs to add attributes (when using `engine: knitr`, there are `attr-*`, but nothing for Jupyter).

**Reply by cderv** [reply]
I think this should be feasible to add `attributes` to the cell enveloppe like we do with `classes`

Probably harder part would be the interaction with `attr-*` from **knitr** in case both are provided. Probably something to handle.

**Comment by mcanouil** [comment]
Closing in favour of #6253
